extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -1161,3 +1318,51 @@ function getFirstInstanceValue_(instances, extractFn) {
   }
   return null;
 }
+
+/**
+ * Maps a sentinel value to an object consisting of the impl to which that
+ * sentinel value belongs and the corresponding message handler for that impl.
+ * @type{!Object<string, !Object>} */
+const fluidListeners = {};
+let listenerForFluidRegistered = false;
+
+/**
+ * @param {!AmpAdNetworkDoubleclickImpl} instance
+ */
+function registerListenerForFluid(instance) {
+  if (!fluidListeners[instance.sentinel]) {
```

Comment:
fluidListeners[instance.sentinel] = fluidListeners[instance.sentinel] || {...};

Reply:
Huh? Not following you here...

Reply:
Done.